boards/arm/wio_terminal/wio_terminal.dts

```diff
@@ -0,0 +1,280 @@
+			read-only;
+		};
+
+		code_partition: partition@4000 {
+			label = "code";
+			reg = <0x4000 DT_SIZE_K(512-16)>;
+			read-only;
+		};
+	};
+};
+
+/* RTL8720D (Wifi/BLE) */
+&sercom0 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <0>;
+	pinctrl-0 = <&sercom0_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portc 25 GPIO_ACTIVE_LOW>;
+};
+
+/* RTL8720D (Wifi/BLE) */
+&sercom1 {
+	status = "okay";
+	compatible = "atmel,sam0-uart";
+	current-speed = <115200>;
+	rxpo = <1>;
+	txpo = <0>;
+	pinctrl-0 = <&sercom1_uart_default>;
+	pinctrl-names = "default";
+};
+
+/* UART, Raspberry Pi connector */
+&sercom2 {
+	status = "okay";
+	compatible = "atmel,sam0-uart";
+	current-speed = <115200>;
+	rxpo = <1>;
+	txpo = <0>;
+	pinctrl-0 = <&sercom2_uart_default>;
+	pinctrl-names = "default";
+};
+
+/* I2C1, Raspberry Pi and Groove connectors */
+&sercom3 {
+	status = "okay";
+	compatible = "atmel,sam0-i2c";
+	clock-frequency = <I2C_BITRATE_FAST>;
+	#address-cells = <1>;
+	#size-cells = <0>;
+	pinctrl-0 = <&sercom3_i2c_default>;
+	pinctrl-names = "default";
+};
+
+/* I2C0, LIS3DH, ATECC608, Raspberry Pi connector */
+&sercom4 {
+	status = "okay";
+	compatible = "atmel,sam0-i2c";
+	clock-frequency = <I2C_BITRATE_HIGH>;
+	#address-cells = <1>;
+	#size-cells = <0>;
+	pinctrl-0 = <&sercom4_i2c_default>;
+	pinctrl-names = "default";
+
+	/* LIS3DH */
+	lis3dh: lis3dh@18 {
+		compatible = "st,lis3dh", "st,lis2dh";
+		status = "okay";
+		reg = <0x18>;
+		irq-gpios = <&portc 21 GPIO_ACTIVE_HIGH>;
+	};
+
+	/* ATECC608A */
+	atecc608a@6a {
+		compatible = "atmel,atecc608";
+		reg = <0x6a>;
+	};
+};
+
+/* SPI, Raspberry Pi connector */
+&sercom5 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <0>;
+	pinctrl-0 = <&sercom5_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portb 1 GPIO_ACTIVE_LOW>;
+};
+
+/* microSD Card */
+&sercom6 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <0>;
+	pinctrl-0 = <&sercom6_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portc 19 GPIO_ACTIVE_LOW>;
+
+	/* microSD Card */
+	sdhc0: sdhc@0 {
+		compatible = "zephyr,sdhc-spi-slot";
+		reg = <0>;
+		spi-max-frequency = <24000000>;
+		mmc {
+			compatible = "zephyr,sdmmc-disk";
+		};
+	};
+};
+
+/* LCD */
+&sercom7 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	dipo = <2>;
+	dopo = <3>;
+	pinctrl-0 = <&sercom7_spi_default>;
+	pinctrl-names = "default";
+	cs-gpios = <&portb 21 GPIO_ACTIVE_LOW>;
+
+	/* LCD */
+	ili9341: ili9341@0 {
+		compatible = "ilitek,ili9341";
+		spi-max-frequency = <24000000>;
+		reg = <0>;
+		cmd-data-gpios = <&portc 6 GPIO_ACTIVE_LOW>;
+		reset-gpios = <&portc 7 GPIO_ACTIVE_LOW>;
+		pixel-format = <ILI9XXX_PIXEL_FORMAT_RGB565>;
+		rotation = <270>;
+		width = <240>;
```

Comment:
I've just tested some GUI code and realized you will want to swap width and height here

Reply:
Good catch! I have just tested with `samples/drivers/display` and it makes no differences.
Are you testing with a sample from Zephyr or something else ?
Will do the modification right now.

Reply:
The issue occurs with samples/subsys/display/lvgl (which I guess is actually taking the width/height information into account vs. the "dumb" driver sample) -- it shows a white screen when the width and height are wrong (in fact I think the app even crashes but I am too lazy to test again).

Reply:
Checked with this sample, yeah I agree. Modification has been done, sample is properly displaying the hello world :-)